src/AWS.Logger.Core/AWSLoggerConfig.cs

```diff
@@ -35,6 +35,22 @@ public class AWSLoggerConfig : IAWSLoggerConfig
         /// </summary>
         public bool DisableLogGroupCreation { get; set; }
 
+        /// <summary>
+        /// Specifies the days to retain events in log groups which are created by this logger, if the one specified by <see cref="LogGroup"/> doesn't already exist
+        /// and <see cref="DisableLogGroupCreation"/> is not <c>true</c>. Requires logs:PutRetentionPolicy permission to apply the retention policy to
+        /// newly created log groups. The default value of <c>null</c> will not apply a retention policy to new log groups.
+        /// <para>
+        /// Note that log groups which already exist will not have this retention policy applied for startup performance reasons.
+        /// </para>
+        /// <para>
+        /// Possible values are can be found in the CloudWatch documentation : https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax
```

Comment:
The code comment is not grammatically correct:
```
Possible values are can be found in the CloudWatch documentation : https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax
```

Notice extra `are can`.

Instead it should read as:
```
Possible values could be found in the CloudWatchLogs API reference at https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax
```